modules/appeals_api/lib/appeals_api/upload_error.rb

```diff
@@ -4,6 +4,7 @@
 
 module AppealsApi
   class UploadError < CentralMail::UploadError
+    # TODO: how does this work?
```

Comment:
Looks like it inherits from StandardError in `lib/central_mail/upload_error.rb` The StatsD code overwrites the constant to attach the appeals namespace to our specific central mail error.

Reply:
Where is 'api.appeals.higher_level_review_upload.fail' defined?